aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -299,30 +299,19 @@ pub struct AggregatorData {
 impl AggregatorData {
     /// Returns a mutable reference to an aggregator with `id` and a `limit`.
     /// If transaction that is currently executing did not initialize it, a new aggregator instance is created.
-    /// The state of the new aggregator instance depends on the `aggregator_enabled` flag.
-    /// If the `aggregator_enabled` flag is true, the new aggregator instance
-    /// is initialized with zero and in a delta state.
-    /// If the `aggregator_enabled` flag is false, the new aggregator instance
-    /// is initialized in the Data state with its latest value.
     /// Note: when we say "aggregator instance" here we refer to Rust struct and
     /// not to the Move aggregator.
     pub fn get_aggregator(
         &mut self,
         id: AggregatorID,
         limit: u128,
-        resolver: &dyn TableResolver,
-        aggregator_enabled: bool,
     ) -> PartialVMResult<&mut Aggregator> {
         let aggregator = self.aggregators.entry(id).or_insert(Aggregator {
             value: 0,
             state: AggregatorState::PositiveDelta,
             limit,
             history: Some(History::new()),
         });
-
-        if !aggregator_enabled {
```

Comment:
Can this break replay?

Reply:
We have earlier added this flag but always set this flag to true in the execution. So, removing this flag won't break backward compatibility.